tests/js/spec/components/__snapshots__/alertLink.spec.jsx.snap

```diff
@@ -11,7 +11,8 @@ exports[`AlertLink renders 1`] = `
   -ms-flex-align: center;
   align-items: center;
   border: 1px dashed;
-  padding: 1em;
+  padding: NaNpx;
```

Comment:
Are we cool with all these NaN values? 🤖 I think it is possible to pass in theme context but maybe not worth it with so many files...

Reply:
I think we should follow up with a PR that fixes this, I'd prefer not having to wrap everything with a ThemeProvider, but rather have a helper func for it